routes/items.test.js

```diff
@@ -81,6 +81,10 @@ describe("/items", () => {
       expect(itemData.items.length).toEqual(1);
       expect(itemData.items).toEqual([items[0]]);
     });
+    it("should return 404 if item does not exist", async () => {
+      const res = await request(server).get("/items/other");
+      expect(res.statusCode).toEqual(404);
+    });
```

Comment:
This isn't a new test. If we want to test that the delete route handler returns a 404 as appropriate, we need to test that path.
```
it("should return 404 if item does not exist", async () => {
      const res = await request(server).delete("/items/other");
      expect(res.statusCode).toEqual(404);
    });
```

Reply:
path you mean URL ?